src/main/java/com/amazon/ion/system/IonTextWriterBuilder_1_1.java

```diff
@@ -0,0 +1,367 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.ion.system;
+
+import com.amazon.ion.IonWriter;
+import com.amazon.ion.Timestamp;
+
+import java.nio.charset.Charset;
+
+/**
+ * The builder for creating {@link IonWriter}s emitting the 1.1 version of the Ion text format.
+ * <p>
+ * Builders may be configured once and reused to construct multiple
+ * objects.
+ * <p>
+ * <b>Instances of this class are not not safe for use by multiple threads
+ * unless they are {@linkplain #immutable() immutable}.</b>
+ *
+ */
+public interface IonTextWriterBuilder_1_1 extends IonWriterBuilder_1_1<IonTextWriterBuilder_1_1> {
+    // TODO add any configuration specific to writing 1.1 text.
+
+    /**
+     * Gets the charset denoting the output encoding.
+     * Only ASCII and UTF-8 are supported.
+     *
+     * @return may be null, denoting the default of UTF-8.
+     *
+     * @see #setCharset(Charset)
+     * @see #withCharset(Charset)
+     */
+    Charset getCharset();
+
+    /**
+     * Sets the charset denoting the output encoding.
+     * Only ASCII and UTF-8 are supported.
+     *
+     * @param charset may be null, denoting the default of UTF-8.
+     *
+     * @see #getCharset()
+     * @see #withCharset(Charset)
+     *
+     * @throws UnsupportedOperationException if this is immutable.
+     */
+    void setCharset(Charset charset);
+
+    /**
+     * Declares the charset denoting the output encoding,
+     * returning a new mutable builder if this is immutable.
+     *
+     * @param charset may be null, denoting the default of UTF-8.
+     *
+     * @return this instance, if mutable;
+     * otherwise a mutable copy of this instance.
+     *
+     * @see #getCharset()
+     * @see #setCharset(Charset)
+     */
+    IonTextWriterBuilder_1_1 withCharset(Charset charset);
+
+    /**
+     * Declares the output encoding to be {@code US-ASCII}.
+     *
+     * @return this instance, if mutable;
+     * otherwise a mutable copy of this instance.
+     */
+    IonTextWriterBuilder_1_1 withCharsetAscii();
+
+    /**
+     * Declares that this builder should minimize system-level output
+     * (Ion version markers and local symbol tables).
+     * <p>
+     * This is equivalent to:
+     * <ul>
+     *   <li>{@link #setIvmMinimizing(IonWriterBuilder.IvmMinimizing)
+     *   setIvmMinimizing}{@code (}{@link IonWriterBuilder.IvmMinimizing#DISTANT DISTANT}{@code )}
+     *   <li>{@link #setLstMinimizing(IonTextWriterBuilder.LstMinimizing)
+     *   setLstMinimizing}{@code (}{@link IonTextWriterBuilder.LstMinimizing#EVERYTHING EVERYTHING}{@code )}
+     * </ul>
+     *
+     * @return this instance, if mutable;
+     * otherwise a mutable copy of this instance.
+     *
+     */
+    IonTextWriterBuilder_1_1 withMinimalSystemData();
```

Comment:
Is IVM minimizing actually a useful option for Ion 1.1? We always have to start with the IVM.
Same thing with the LST minimizing. If we're just minimizing the symbol table, then this seems redundant with `SymbolInliningStrategy`. However, if we're minimizing the macro table as well... does that mean we're doing write-time evaluation of macros so that we can write the data without E-Expressions?

I don't think there's necessarily any action item from this comment, but it's something to keep in mind.

Reply:
I think IVM minimization is potentially applicable (though I'm not sure it's that useful, even in Ion 1.0), because it allows us to de-duplicate multiple IVMs that occur in a row.

For LST/macro table minimization, I see this option as on by default (because they're almost never needed explicitly), but could be disabled. Let's follow up on this before finalizing Ion 1.1 to make sure we don't have options included in the builder that we never intend to support, or that are made redundant by other options that we add.